docs/index.md

```diff
@@ -10,6 +10,11 @@ title: AddressBook Level-3
 
 **ClientConnect is a desktop application for managing your clients' details and insurance packages.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
+Add Tags with varying levels of priorities to keep track of utmost important tasks to utilize your time efficiently,
+sort your clients by priority so you won't miss out urgent tasks to be done, conveniently import and export your client contacts, and easily find the client details you want through our `find` command. 
```

Comment:
Shouldn't this be in the same line as the above?

Reply:
oh didn't realise they actually created a new line, thanks for the notice sam, could you help me change that?

Reply:
Oh man didn't realise a new line was made, thanks for noticing sam